src/DashboardService.Business/Board/GetBoardCommand.cs

```diff
@@ -15,19 +19,30 @@ public class GetBoardCommand : IGetBoardCommand
   private readonly IBoardRepository _boardRepository;
   private readonly IBoardResponseMapper _boardResponseMapper;
   private readonly IResponseCreator _responseCreator;
+  private readonly IGroupInfoMapper _groupInfoMapper;
 
   public GetBoardCommand(
     IBoardRepository repository,
     IBoardResponseMapper boardResponseMapper,
-    IResponseCreator responseCreator)
+    IResponseCreator responseCreator,
+    IGroupInfoMapper groupInfoMapper)
   {
     _boardRepository = repository;
     _boardResponseMapper = boardResponseMapper;
     _responseCreator = responseCreator;
+    _groupInfoMapper = groupInfoMapper;
   }
 
-  public Task<OperationResultResponse<BoardResponse>> ExecuteAsync(Guid id, GetBoardFilter filter)
+  public async Task<OperationResultResponse<BoardResponse>> ExecuteAsync(Guid id, GetBoardFilter filter, CancellationToken ct)
   {
-    throw new NotImplementedException();
+    DbBoard dbBoard = await _boardRepository.GetAsync(id, filter, ct);
+
+    if (dbBoard is null)
+    {
+      return _responseCreator.CreateFailureResponse<BoardResponse>(HttpStatusCode.NotFound);
+    }
+
+    return new OperationResultResponse<BoardResponse>(
+      body: _boardResponseMapper.Map(dbBoard, (x) => _groupInfoMapper.Map(x)));
```

Comment:
You can write only "_groupInfoMapper.Map"

Reply:
++